bin/boot_docker

```diff
@@ -141,7 +145,8 @@ then
 	# generate random secret key for secure cookies
 	docker run -d --name "$CONTAINER_NAME" -p 80:80 -p 443:443 -h "$PROD_HOSTNAME" -v "$PROJECT_DIR:/home/app/webapp:rw" \
 	    -v "$PROJECT_DIR/data:/home/app/webapp/data:rw" -v "$PROJECT_DIR/log/nginx:/var/log/nginx:rw" -e PASSENGER_APP_ENV="$PASSENGER_APP_ENV" \
-	    -e MONGO_LOCALHOST="$MONGO_LOCALHOST" -e PROD_DATABASE_PASSWORD="$PROD_DATABASE_PASSWORD" -e SERVICE_ACCOUNT_KEY="$SERVICE_ACCOUNT_KEY" \
+	    -e MONGO_LOCALHOST="$MONGO_LOCALHOST" -e MONGO_INTERNAL_IP="$MONGO_INTERNAL_IP" -e PROD_DATABASE_PASSWORD="$PROD_DATABASE_PASSWORD" \
+	    -e GCP_NETWORK_NAME=$GCP_NETWORK_NAME -e GCP_SUB_NETWORK_NAME=$GCP_SUB_NETWORK_NAME -e SERVICE_ACCOUNT_KEY="$SERVICE_ACCOUNT_KEY" \
```

Comment:
Could you point me to where these two new environment variables -- GCP_NETWORK_NAME and GCP_SUB_NETWORK_NAME -- are defined? I don't see them among the command-line options defined above, so my hunch is this is a bug.

Assuming that's so, once those CLI options are added, would it also be worth documenting them in the README?

Reply:
Yes, they will need to be added to our vault configurations.  As far as the README is concerned, it would only affect installations where the MongoDB VM is not on the default network, which is likely not something other places would do.  I'll see if there's a place where it makes sense to call this out, regardless.